Sources/Sentry/SentryCoreDataTracker.m

```diff
@@ -54,6 +67,14 @@ - (BOOL)managedObjectContext:(NSManagedObjectContext *)context
                                           description:[self descriptionForOperations:operations
                                                                            inContext:context]];
 
+            [SentryLog
+                logWithMessage:
+                    [NSString
+                        stringWithFormat:@"SentryCoreDataTracker automatically "
+                                         @"created a new span with description: %@, operation: %@",
```

Comment:
```suggestion
                                         @"started a new span with description: %@, operation: %@",
```

"created" could be interpreted as a combined start/stop to produce a finished span, i think "started" would be clearer.